configuration/configuration.go

```diff
@@ -202,6 +215,16 @@ func (c *ConfigurationData) setConfigDefaults() {
 	c.v.SetDefault(varCacheControlFilters, "max-age=86400")
 	c.v.SetDefault(varCacheControlUsers, "max-age=2")
 	c.v.SetDefault(varCacheControlCollaborators, "max-age=2")
+
+	// Cache control values for a single resource
+	c.v.SetDefault(varCacheControlWorkItem, "private,max-age=120")
```

Comment:
Reduce to 2 for WorkItem and Iteration(i think iteration contain a count of wis etc), and increase to 120 for User.